answer.txt

```diff
@@ -0,0 +1,12 @@
+• What is a gitignore file and what should it include for a Java project?
+
+A. It is a hidden file which stores the file names or file formats which git should ignore for checking changes during staging/ modifying the file contents.
+For a typical java project it should include .class files.
+
+• What does fast-forward in Git mean?
+A. When we merge a branch which has more commits than another branch, upon merging the base branch gets updated in a 'fast-forward' method.
```

Comment:
For this too a better explanation might be helpful.

Reply:
Okay